content/api/entity-reference/fundraising/index.html

```diff
@@ -68,7 +68,7 @@
             <tr>
                 <td>inactive</td>
                 <td>boolean</td>
-                <td>Indicates whether the appeal is active.</td>
+                <td>Indicates whether the appeal is active. This is a computed field based on <code>start_date</code> and <code>end_date</code> properties.</td>
```

Comment:
basically, if there is an an end date and todays date is outside of the start and end date range its inactive. Otherwise its active